File: src/work/mojamoja/logging/Logging.java
package work.mojamoja.logging;

import org.bukkit.plugin.java.JavaPlugin;

public class Logging extends JavaPlugin {
    @Override
    public void onEnable() {
        System.out.println("Logging enabled.");
    }

    @Override
    public void onDisable() {
        System.out.println("Logging disabled.");
    }
}

File: src/work/mojamoja/logging/Logging.kt
package work.mojamoja.logging

import org.bukkit.plugin.java.JavaPlugin

class Logging: JavaPlugin() {
    override fun onEnable() {
        logger.info("Logging enabled.")
        server.pluginManager.registerEvents(Cutter(), this)

        // config.ymlが存在しない場合、デフォルトのconfig.ymlをplugins/Logging以下に作成する。
        this.saveDefaultConfig()
    }

    override fun onDisable() {
        logger.info("Logging disabled.")
    }
}